python_exercises/question_directory/boolean_operators.py

```diff
@@ -0,0 +1,41 @@
+# Slide link: https://kathrinschuler.github.io/slide-python-intro/#/11/4
+
+# TASKS is a list of lists. Each sublist follows this structure:
+# [
+#   <string of background information to print>,
+#   <string of question text>,
+#   <list of answer keywords>,
+#   <dictionary of prerequisite conditions>
+# ]
+
+"""
+empty object for copy-paste:
+[
+    "", # background info
+    "", # question
+    [], # list of required keywords
+    {}, # dict of variables to set up before question
+],
+"""
+
+BLURB = ""
+TASKS = [
+    [
+        "",  # background info
```

Comment:
I think it is worth it to copy the same background info as the next questions:

"" -> "\ngoogle_up = True\nfacebook_up = True\n"